docs/configexamples/fwall-and-bridge.rst

```diff
@@ -0,0 +1,496 @@
+:lastproofread: 2024-09-11
+
+Bridge and firewall example
+---------------------------
+
+Scenario and requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This example shows how to configure a VyOS router with bridge and firewall
+rules.
+
+Three non vlan-aware bridges are going to be configured, and each one has its
+own requirements.
+
+* Bridge br0:
+   * Isolated layer 2 bridge.
+   * Accept only IPv6 communication whithin the bridge.
+
+* Bridge br1:
+   * Drop all DHCP discover packets.
+   * Accept all ARP packets.
+   * Within the bridge, accept only new IPv4 connections from host 10.1.1.102
+   * Drop all other IPv4 connections.
+   * Drop all IPv6 connections.
+   * Accept access to router itself.
+   * Allow connections to internet
+   * Drop connections to other LANs.
+
+* Bridge br2:
+   * Accept all DHCP discover packets.
+   * Accept only DHCP offers from valid server and|or trusted bridge port.
+   * Accept all ARP packets.
+   * Accept all IPv4 connections.
+   * Drop all IPv6 connections.
+   * Deny access to the router.
+   * Allow connections to internet.
+   * Allow connections to bridge br1.
+
+Configuration
+^^^^^^^^^^^^^
+
+Bridges and interfaces configuration
+""""""""""""""""""""""""""""""""""""
+
+First, we need to configure the interfaces and bridges:
+
+.. code-block:: none
+
+  # Brige br0
+  set interfaces bridge br0 description 'Isolated L2 bridge'
+  set interfaces bridge br0 member interface eth1
+  set interfaces bridge br0 member interface eth2
+  set interfaces ethernet eth1 description 'br0'
+  set interfaces ethernet eth2 description 'br0'
+
+  # Bridge br1:
+  set interfaces bridge br1 address '10.1.1.1/24'
+  set interfaces bridge br1 description 'L3 bridge br1'
+  set interfaces bridge br1 member interface eth3
+  set interfaces bridge br1 member interface eth4
+  set interfaces ethernet eth3 description 'br1'
+  set interfaces ethernet eth4 description 'br1'
+
+  # Bridge br2:
+  set interfaces bridge br2 address '10.2.2.1/24'
+  set interfaces bridge br2 description 'L3 bridge br2'
+  set interfaces bridge br2 member interface eth5
+  set interfaces bridge br2 member interface eth6
+  set interfaces bridge br2 member interface eth7
+  set interfaces ethernet eth5 description 'br2 - Host'
+  set interfaces ethernet eth6 description 'br2 - Trusted DHCP Server'
+  set interfaces ethernet eth7 description 'br2'
+
+Bridge firewall configuration
+"""""""""""""""""""""""""""""
+
+In this section, we are going to configure the firewall rules that will be used
+in bridge firewall, and will control the traffic within each bridge.
+
+We are going to use custom firewall rule-sets, one for each bridge that will
```

Comment:
```suggestion
We are going to use custom firewall rulesets, one for each bridge that will
```

If it's spelled without a hyphen elsewhere, we need to be consistent here as well.